tests/unit/workflows/dotnet_clipackage/test_actions.py

```diff
@@ -176,7 +176,7 @@ def test_build_package_arguments(self):
                 "--msbuild-parameters",
                 "--runtime linux-x64",
                 "--framework",
-                "netcoreapp2.1",
+                "net6.0",
```

Comment:
This example and the actual option don't match: https://github.com/aws/aws-lambda-builders/pull/492/files#diff-a33133aa75d3cbb0bf53e9c46bc3bb116a3f94fbd343933949990fe3c4eef5f1R44

should we get them to be the same?